go/merge/three_way_keyval_test.go

```diff
@@ -64,18 +72,22 @@ var (
 
 type ThreeWayMergeSuite struct {
```

Comment:
Sorry... could you leave the stuff that applies to both set and keyval tests in a three_way_test.go file? I.e. ThreeWayMergeSuite and friends.